content/tutorials/pulumi-deployments-click-to-deploy/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title_tag: Create Infrastructure with Pulumi Deployments Click-to-Deploy | Learn Pulumi
+title: "Create Infrastructure with Pulumi Deployments Click-to-Deploy"
+layout: single
+description: |
+    Create and deploy infrastructure in the browser with Pulumi Deployments and the New Project Wizard.
+meta_desc: In this step-by-step tutorial, learn how to create and manage cloud infrastructure using Pulumi Deployments, the New Project Wizard, and GitHub integration.
+meta_image: meta.png
+weight: 50
+summary: |
+    In this tutorial, you'll learn how to create, deploy, and manage cloud infrastructure using Pulumi Deployments with click-to-deploy. You'll start by using the New Project Wizard to set up and launch infrastructure from your browser, selecting a template to get started quickly. Additionally, you'll configure the Pulumi GitHub App to support additional deployment triggers, including push-to-deploy.
+youll_learn:
+  - How to use the New Project Wizard to create and deploy a Pulumi project from your browser
+  - How to install and configure the Pulumi GitHub App
+  - How to trigger and manage deployments directly from the Pulumi Cloud console using click-to-deploy
+prereqs:
+    - Completion of the AWS [Getting Started](/docs/clouds/aws/get-started/) guide or familiarity with the basics of the Pulumi workflow
+    - A [Pulumi Cloud account](https://app.pulumi.com/signup)
+    - AWS credentials configured
```

Comment:
My first question on reading this would be "configured how"? Local configuration won't matter of course, since we won't be running locally. We should explain what "configured" means or just remove it, and leave it at "AWS credentials", or "An AWS acccount, access key ID, and secret access key". (The last version also mentioned needing an AWS account, which we seem to have removed.)